test/mobile/lightweight_dispatch/build.sh

```diff
@@ -0,0 +1,54 @@
+# This script should be called from .jenkins/pytorch/build.sh. Assuming we are at pytorch source root directory.
```

Comment:
missing a shebang? Also consider `set -ex` (e for quit on error, x for printing commands as they are run).